### PR TITLE
Added new track

### PR DESCRIPTION
**Pull request by nampi**

### CODE REVIEW

**Comment by nampi** [comment]
Youtube videos are in progress (waiting for text approval).

File: _posts/2015-02-04-2.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title:  "Taking a Photo"
+date:   2015-02-04 10:00:02
+answerrostag: "simulation_challenge"
+track: [simulation]
+---
+
+[comment]: <> (TODO: replace answerrostag and track)
```
Comment (tfoote) [inline _posts/2015-02-04-2.markdown:9]: A new track would be great. Go ahead and pick something. We can always change it later.

**Comment by nampi** [comment]
@tfoote I have updated this pull request. Could you please review it?

File: _posts/2015-02-04-2.markdown [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+8\. Run `ls` command.
+{% highlight sh %}
+ls
+{% endhighlight %}
+
+You will see the new files.
+
+9\. Open the image file.
+{% highlight sh %}
+ristretto tutorial00.jpg
```
Comment (tfoote) [inline _posts/2015-02-04-2.markdown:89]: I don't have this command available. `eog` might be better assuming ubuntu or `gnome-open` assuming gnome.

**Comment by tfoote** [comment]
Reading it through it looks good. I made one comment about the image preview command. I'll merge it and you can go back and review that.

**Pull request merged**